sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config.adoc

```diff
@@ -26,34 +28,27 @@ During this time, however, the API always remains backwards compatible!
 `server` (optional) defines the location to `{sechub}` server.
 Information can be used by client implementations.
 
-NOTE: When using REST calls directly you do not need the information inside
-      configuration because you use server URL in every call...
+NOTE: This is optional parameter only interesting for the client to pass the URL.
+      Every REST call contains the server URL already.
 
 ==== Project
 `project` (optional) defines the project to use. Information can be used by
 client implementations.
 
-NOTE: When using REST calls directly you do not need the information inside
-      configuration because you use project id inside URL.
+NOTE: When using project id related REST calls, you do not need the project id inside the configuration.
 
 [[sechub-config-code-scan]]
 ==== Code scan
-`codeScan` (optional) defines the code scan settings.
-
-===== Upload sources from file system
-Define `fileSystem` child entry and setup folders to use for code scanning by
-`folders` sub entry.
+`codeScan` (optional) defines the settings for code scans.
 
-All source code files inside these folders will be automatically zipped and uploaded by `{sechub}` client
-to server side for ongoing source analysis.
+===== Upload source files from file system
+This section applies only to the {sechub} client.
 
-Pathes must be defined relative from caller location - for example
-on a Jenkins Build Server this is normally the root folder of repository.
+All source code files inside the defined folders will be automatically zipped and uploaded by `{sechub}` client
+to server side for source code analysis.
 
-NOTE: The configuration parts inside `fileSystem` entry are interpreted by
-      `{sechub}` client! When you directly use the REST API you do not need
-      to define this data but you have to upload sources as a `zip`file by
-      your own over REST!
+Pathes must be defined relative to caller location - for example
```

Comment:
```suggestion
Paths must be defined relative to caller location - for example
```

Reply:
That's a good one - I have changed this in multiple files.